nucypher/config/node.py

```diff
@@ -396,7 +397,9 @@ def validate(self, config_root: str, no_registry=False) -> bool:
 
         for field, path in filepaths.items():
             if not os.path.exists(path):
-                message = 'Missing configuration directory {}.'
+                message = 'Missing configuration file or directory: {}.'
+                if 'registry' in path:
```

Comment:
Maybe a comment here describing to the reader *why* we expect 'registry' if they perhaps meant `--federated-only`?

Reply:
:heavy_check_mark: